### PR TITLE
Initialize frontend setup

### PR DESCRIPTION
**Pull request by TzviPM**
This PR adds support for:
- compilation:
  - webpack 4
  - babel 7
  - typescript 2.7
- linting
  - eslint
  - typescript
  - prettier
- testing
  - jest

### CODE REVIEW

File: Frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# graphql-api-base-frontend
+
+A frontend for `graphql-api-base`.
```
Comment (ignacio-chiazzo) [inline Frontend/README.md]: mention web here

File: Frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Getting started
+
+Install
+
+```bash
```
Comment (ignacio-chiazzo) [inline Frontend/README.md]: I personally prefer inline to make it shorter. (Also in bold)
**Install:** `yarn`

**Review by ignacio-chiazzo — APPROVED** [review]

**Comment by ignacio-chiazzo** [comment]
Isn't it too many dependencies?

File: Frontend/README.md (hunk 2 above)
Comment (TzviPM) [reply Frontend/README.md]: Nice

File: Frontend/README.md (hunk 1 above)
Comment (TzviPM) [reply Frontend/README.md]: 👍🏼

**Comment by TzviPM** [comment]
I'm thinking of a way to manage all this from a single dependency. Something like create-react-app does, only more generic. The problem I ran into so far is how to manage all of the configuration files. I'm not sure if they should be all one file, or many files to be consistent with the specs for each tool.

**Comment by TzviPM** [comment]
After thinking about his some more, I think the dependencies are okay. I like the fact that the tooling is all explicit, so that people can make configuration changes if they want to.

**Pull request merged**